pkg/synopsysctl/cmd_get.go

```diff
@@ -176,9 +218,16 @@ func init() {
 	rootCmd.AddCommand(getCmd)
 
 	// Add Commands
-	getCmd.AddCommand(getBlackduckCmd)
+	getBlackduckCmd.Flags().StringVarP(&getOutputFormat, "output", "o", getOutputFormat, "kubectl's output format")
```

Comment:
Let's not specify kubectl in the help.  Let's just print the information that we support.  We are defining the output formats anyway.